common/src/group.ts

```diff
@@ -45,17 +45,8 @@ export type GroupLink = {
   userId?: string
 }
 
-export function groupPath(
-  groupSlug: string,
-  subpath?:
-    | 'edit'
-    | 'markets'
-    | 'about'
-    | typeof GROUP_CHAT_SLUG
-    | 'leaderboards'
-    | 'posts'
-) {
-  return `/group/${groupSlug}${subpath ? `/${subpath}` : ''}`
+export function groupPath(groupSlug: string) {
+  return `/group/${groupSlug}`
 }
```

Comment:
Don't we still support linking to subpages this way? Seems like we should keep this.

Reply:
I guess that right now this would only be for leaderboards. It's ok to break this, but what if we have other tabs in the future for groups? I'd only do this if it was interfering with the other url params, which I don't think that it would be?

Reply:
Also, kind of minor, but I don't like the pattern of using tab=blah instead of adding a path /blah. From the user's perspective, the latter is way cleaner and easier to parse.
I think this is partly on NextJS for making this hard.